updatecli/updatecli.d/update-plugins.ps1

```diff
@@ -25,14 +33,13 @@ foreach ($dependency in $dependencies) {
   $artifact = $dependency.artifactId
   $oldVersion = $dependency.version
   $plugin = "${artifact}:${oldVersion}"
-  [string] $output = & $java -jar "$PluginManagerJar" --no-download --available-updates --jenkins-version "$JenkinsVersion" --plugins $plugin
-  if ($null -ne $output -and $output -inotlike '*No available updates*') {
+  [string] $output = & $java -jar "$pluginManagerJar" --no-download --available-updates --output txt --jenkins-version "$JenkinsVersion" --plugins $plugin
+  if ($output -ne $plugin) {
```

Comment:
nice this will always output the plugin either with the `email-ext:2.86` or when updates are available `email-ext:2.87` making comparison much easier.